sql/mob_family_mods.sql

```diff
@@ -240,12 +238,11 @@ INSERT INTO `mob_family_mods` VALUES (72,52,30,1); -- ROAM_RATE: 30
 INSERT INTO `mob_family_mods` VALUES (72,68,20,0); -- EVA: 20
 
 -- Corse
-INSERT INTO `mob_family_mods` VALUES (74,29,25,0);   -- MDEF: 25
```

Comment:
Are you sure this should be removed? I saw notes that banish should lower an innate magic defense corses have, similar to the piercing defense lowered for skeletons.

Reply:
It shouldn't need MDEF bc of our SDT changes that were implemented - should be taken care of